### ISSUE TITLE
Investigate automated accessibility testing

### ISSUE THREAD

**Issue by rahulporuri** [fossunited/fossunited#790]
fossunited.org is a work-in-progress. While addressing existing accessibility issues is valuable, it will be great to also investigate how future code changes that cause accessibility regression can be caught and reported through automated testing.

We would like to understand what tools are available to setup such automated testing, experiment with a few such automated tests, and understand how easy/difficult it will be to setup and maintain such tests and infra.

Ref https://slack.engineering/automated-accessibility-testing-at-slack/
